NAMESPACE

```diff
@@ -3,3 +3,7 @@ import(RJSONIO)
 importFrom(utils, packageDescription)
 
 exportPattern("^[[:alpha:]]+")
+
+S3method("pbGetDevices", "default")
```

Comment:
Also, S3method() becomes important when a package is not attached, but only loaded, which happens when other packages import such functions.  Typical symptoms are hard to track errors.